connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/UplynkAdIntegration.kt

```diff
@@ -0,0 +1,16 @@
+package com.theoplayer.android.connector.uplynk
+
+import com.theoplayer.android.api.THEOplayerView
+import com.theoplayer.android.api.ads.ServerSideAdIntegrationController
+import com.theoplayer.android.api.ads.ServerSideAdIntegrationHandler
+import com.theoplayer.android.connector.uplynk.internal.InternalUplynkAdIntegration
+import com.theoplayer.android.connector.uplynk.internal.UplynkSsaiDescriptionConverter
+
+class UplynkAdIntegration(
```

Comment:
Why is this class in the public API? It seems like it doesn't *need* to be exposed?

I suggest we delete it, and use `InternalUplynkAdIntegration` directly.